ide-migration/server/migration/api/migration-service.mjs

```diff
@@ -80,13 +80,14 @@ export class MigrationService {
 
     createHdiFile(workspaceName, project, hdiConfigPath, deployables) {
         const projectName = project.getName();
+        const groupOrContainerHdiValue = projectName.toUpperCase();
```

Comment:
Is the **Delivery Unit** name available here? It would be better to set the container and group to the delivery unit instead of the project (root folder) name.

Reply:
It is available but are you sure we should set it to the group and container fields? What happens if there are multiple projects with an HDI file generated for each and they have the same group/container name?

Reply:
I'm fine with this format and I'll update the PR accordingly